src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -85,73 +85,90 @@ namespace
         ParamMapIdImageSampling,
         ParamMapIdLighting,
         ParamMapIdPathTracer,
+        ParamMapIdSPPM,
```

Comment:
The ParamMapIdSPPM is necessary so that the SPPM controls can have their own rollout, like the Path Tracing controls have. Otherwise one would have to bunch them all together with the Path Tracing ones, making it more confusing. I don't think it breaks compatibility with old scenes, I test loaded the old cookie scene without issues.